js/cryptocom.js

```diff
@@ -0,0 +1,1207 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ArgumentsRequired, ExchangeError, InsufficientFunds, DDoSProtection, InvalidNonce, PermissionDenied, BadRequest, BadSymbol } = require ('./base/errors');
+const Precise = require ('./base/Precise');
+
+module.exports = class cryptocom extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'cryptocom',
+            'name': 'Crypto.com',
+            'countries': [ 'MT' ],
+            'version': 'v2',
+            'rateLimit': 10, // 100 requests per second
+            'has': {
+                'cancelAllOrders': true,
+                'cancelOrder': true,
+                'CORS': false,
+                'createOrder': true,
+                'fetchCurrencies': false,
+                'fetchBalance': true,
+                'fetchBidsAsks': false,
+                'fetchClosedOrders': 'emulated',
+                'fetchDepositAddress': true,
+                'fetchDeposits': true,
+                'fetchFundingFees': false,
+                'fetchFundingHistory': false,
+                'fetchFundingRate': false,
+                'fetchFundingRates': false,
+                'fetchIsolatedPositions': false,
+                'fetchMarkets': true,
+                'fetchMyTrades': true,
+                'fetchOHLCV': true,
+                'fetchOpenOrders': true,
+                'fetchOrder': true,
+                'fetchOrders': true,
+                'fetchOrderBook': true,
+                'fetchPositions': false,
+                'fetchStatus': false,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchTime': false,
+                'fetchTrades': true,
+                'fetchTradingFee': false,
+                'fetchTradingFees': false,
+                'fetchTransactions': false,
+                'fetchWithdrawals': false,
+                'setLeverage': false,
+                'setMarginMode': false,
+                'withdraw': false,
```

Comment:
Looks like they have a `withdraw` method: https://exchange-docs.crypto.com/spot/index.html#private-create-withdrawal

Reply:
They also have fetchDeposits, fetchWithdrawals and transfer.

Reply:
yeah...

Reply:
I think we can merge it for now without transfer and then add it once we add support for margin and derivatives (currently it only supports spot)

Reply:
We should still add depositing and withdrawing though.

Reply:
Ah, nvm, now i see them added.